src/components/language-picker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from 'react'
+
+import styled from 'styled-components/macro'
+import { useTranslation } from 'react-i18next'
+
+const LanguagePickerButton = styled.button`
+  color: ${props => props.theme.colors.text};
+  font-size: ${props => props.theme.sizes.buttonText};
+  font-weight: 500;
+  border: none;
+  border-radius: 8px;
+  background-color: transparent;
+  padding: 8px;
+  margin: 8px;
+  min-width: 50px;
+  transition: background 200ms linear;
+  cursor: pointer;
+  outline: none;
+
+  &:hover {
+    background-color: ${props => props.theme.colors.primaryLight};
+  }
+`
+
+export const LanguagePicker: React.FC = props => {
+  const { i18n } = useTranslation()
+
+  const otherLanguage = i18n.language === 'fr' ? 'en' : 'fr'
```

Comment:
I'd suggest to use the languages in `package.json` -> `supportedLanguages` and shift the index instead of having any language codes defined in the code

Reply:
but we're still constrained to two languages using a toggle, which is why I figured it wouldn't hurt to hardcode it, but I don't mind

Reply:
done, but using i18n still, package json should be used sparingly